docs/standard/analyzers/api-analyzer.md

```diff
@@ -0,0 +1,86 @@
+---
+title: .NET API analyzer
+description: Learn how the .NET API Analyzer can help detect deprecated APIs and platform compatibility issues.
+author: oliag
+ms.author: mairaw
+manager: 
+ms.date: 01/12/2018
+ms.topic: article
+ms.prod: .net
+ms.technology: dotnet-standard
+---
+# .NET API analyzer
+
+The .NET API Analyzer is a Roslyn analyzer that discovers potential compatibility risks for C# APIs on different platforms and detects calls to deprecated APIs. It can be useful for all C# developers at any stage of development.
+
+API Analyzer comes as a NuGet package [Microsoft.DotNet.Analyzers.Compatibility](https://www.nuget.org/packages/Microsoft.DotNet.Analyzers.Compatibility/) and after you reference it in a project, it automatically monitors the code and indicates problematic API usage. You can also get suggestions on possible fixes by clicking on the light bulb. The drop-down menu includes an option to suppress the warnings.
+
+> [!NOTE]
+> The .NET API analyzer is still a pre-release version.
+
+## Discovering deprecated APIs
+
+**What are deprecated APIs?**
+
+The .NET family is a set of large products that are constantly upgraded to better serve customer needs. It is natural to deprecate some APIs and replace them with new ones. One way to inform that an API is deprecated and should not be used is to mark it with the `[Obsolete]` attribute. The disadvantage of this approach is that there is only one diagnostic ID for all obsolete APIs (for C#, [CS0612](../../csharp/misc/cs0612.md)) that leads to one description for all cases and ability to suppress warnings for either all cases or none of them. The API Analyzer uses API-specific error codes that begin with DE (which stands for Deprecation Error), which allows control over the display of individual warnings. To avoid confusion with the `[Obsolete]` attribute, this concept is called deprecation.
```

Comment:
For those who don't know what the Obsolete attribute is, a link might be better here: \[Obsolete](xref:System.ObsoleteAttribute)
ability --> the ability

for a later revision (no need to do it here), you might note that deprecated APIs are not necessarily marked with Obsolete.